app/components/entity_mentions/comments/fetching.py

```diff
@@ -0,0 +1,209 @@
+        created_at=cast(dt.datetime, comment.submitted_at),
+        html_url=comment.html_url,
+        color=STATE_TO_COLOR.get(comment.state),
+        kind="Review",
+    )
+
+
+def _prettify_suggestions(comment: PullRequestReviewComment) -> str:
+    suggestions = [
+        c for c in extract_codeblocks(comment.body) if c.lang == "suggestion"
+    ]
+    body = comment.body
+    if not suggestions:
+        return body
+
+    start = cast(int | None, comment.original_start_line)
+    end = cast(int, comment.original_line)
+    hunk_size = end - (end if start is None else start) + 1
+    hunk_as_deleted_diff = "\n".join(
+        ("-" + line[1:] if line[0] == "+" else line)
+        for line in comment.diff_hunk.splitlines()[-hunk_size:]
+    )
+
+    for sug in suggestions:
+        suggestion_as_added_diff = f"{hunk_as_deleted_diff}\n" + "\n".join(
+            f"+{line}" for line in sug.body.splitlines()
+        )
+        body = body.replace(
+            _make_crlf_codeblock("suggestion", sug.body.replace("\r\n", "\n")),
+            _make_crlf_codeblock("diff", suggestion_as_added_diff),
+            1,
+        )
+    return body
+
+
+def _make_crlf_codeblock(lang: str, body: str) -> str:
+    # GitHub seems to use CRLF for everything...
+    return f"```{lang}\n{body}\n```".replace("\n", "\r\n")
+
+
+async def _get_pr_review_comment(entity_gist: EntityGist, comment_id: int) -> Comment:
+    owner, repo, _ = entity_gist
+    comment = (
+        await gh.rest.pulls.async_get_review_comment(owner, repo, comment_id)
+    ).parsed_data
+    return Comment(
+        author=_make_author(comment.user),
+        body=_prettify_suggestions(comment),
+        entity=await entity_cache.get(entity_gist),
+        entity_gist=entity_gist,
+        created_at=comment.created_at,
+        html_url=comment.html_url,
+        kind="Review comment",
+    )
+
+
+async def _get_event(entity_gist: EntityGist, comment_id: int) -> Comment:
+    owner, repo, entity_no = entity_gist
+    event = (await gh.rest.issues.async_get_event(owner, repo, comment_id)).parsed_data
+    if event.event not in SUPPORTED_EVENTS.keys() | ENTITY_UPDATE_EVENTS:
+        body = f":ghost: Unsupported event: `{event.event}`"
+    elif event.event == "review_requested":
+        # Special-cased to handle requests for both users and teams
+        if event.requested_reviewer:
+            reviewer = event.requested_reviewer.login
+        else:
+            assert event.requested_team
+            # Throwing in the org name to make it clear that it's a team
+            reviewer = f"{config.GITHUB_ORG}/{event.requested_team.name}"
+        body = SUPPORTED_EVENTS[event.event].format(reviewer=reviewer)
+    elif event.event in ENTITY_UPDATE_EVENTS:
+        entity = await entity_cache.get(entity_gist)
+        body = f"{event.event.capitalize()} the {entity.kind}"
+        if event.lock_reason:
+            body += f"\nReason: `{event.lock_reason or 'no reason'}`"
```

Comment:
`reason` is repeated twice here.
```suggestion
            body += f"\nReason: `{event.lock_reason or 'none'}`"
```
or
```suggestion
            body += f"\nReason: `{event.lock_reason or 'unspecified'}`"
```

Reply:
I'll go with `unspecified`, thanks.